.pre-commit-config.yaml

```diff
@@ -65,3 +65,9 @@ repos:
     hooks:
         - id: rapids-dependency-file-generator
           args: ["--clean"]
+  - repo: https://github.com/shellcheck-py/shellcheck-py
+    rev: v0.10.0.1
+    hooks:
+      - id: shellcheck
+        args: ["--severity=warning"]
+        files: ^ci/
```

Comment:
Could you expand this a bit? I'd really like to see it cover stuff like https://github.com/rapidsai/cugraph/blob/branch-25.04/datasets/get_test_data.sh and https://github.com/rapidsai/cugraph/blob/branch-25.04/scripts/dask/run-dask-process.sh too

If you want to just remove this exclusion and have it cover all shell scripts in this round, I don't mind that making the PR larger.

That'd also be one less repo to have to modify in the second round of this that covers `build.sh` scripts.